src/app/public/modules/fluid-grid/column.component.ts

```diff
@@ -27,7 +27,7 @@ export class SkyColumnComponent implements OnInit, OnChanges {
   public screenLarge: number;
 
   @HostBinding('class')
-  private classnames: string;
+  public classnames: string;
```

Comment:
Why does this need to be public?

Reply:
TypeScript didn't like that it wasn't being used anywhere. It's "technically" a public property since it gets shared with the host template.